src/ansys/mapdl/core/mapdl_geometry.py

```diff
@@ -44,7 +77,7 @@ def merge_polydata(items):
     return pv.wrap(afilter.GetOutput())
 
 
-def get_elements_per_area(resp):
+def get_elements_per_area(resp: str) -> List[int]:
```

Comment:
```suggestion
def get_elements_per_area(resp: str) -> Union[List[List[int]], List[Tuple[int, int]]:
```

It could be interesting to change the [L113](https://github.com/ansys/pymapdl/blob/feat/homogenizing-geometry-API/src/ansys/mapdl/core/mapdl_geometry.py#L113) to replace the list by a tuple to fix the return type to `List[Tuple[int, int]]` which has the advantage of having a fixed size. However this could be considered as a breaking change